src/components/Scroll.js

```diff
@@ -21,18 +38,31 @@ class Scroll extends Component {
 
   static propTypes = {
     children: PropTypes.node.isRequired,
+    arrayLength: PropTypes.number.isRequired,
   }
 
   render() {
-    return (
-      <div className="OuterDiv">
-        <button className="leftbutton scroll-button" onClick={this.scrollLeft}><FontAwesomeIcon icon="angle-left" /></button>
-        <div className="scrolling-wrapper-flexbox" ref={this.inputRef}>
-          {this.props.children}
+    let result = <div className="OuterDiv" />;
+    if ((this.props.arrayLength * 170) < (this.state.width - 250)) {
+      result = (
+        <div className="OuterDiv">
+          <div className="scrolling-wrapper-flexbox" ref={this.inputRef}>
+            {this.props.children}
+          </div>
+        </div>
+      );
+    } else {
+      result = (
+        <div className="OuterDiv">
+          <button className="leftbutton scroll-button" onClick={this.scrollLeft}><FontAwesomeIcon icon="angle-left" /></button>
+          <div className="scrolling-wrapper-flexbox" ref={this.inputRef}>
```

Comment:
`this.inputRef` should be named something else, since it isn't an input. I know this was added long ago, but this should still be changed.